examples/chip-tool/commands/common/Commands.cpp

```diff
@@ -70,16 +70,32 @@ int Commands::Run(NodeId localId, NodeId remoteId, int argc, char ** argv)
     err = mController.Init(localId, initParams);
     VerifyOrExit(err == CHIP_NO_ERROR, ChipLogError(Controller, "Init failure! Commissioner: %s", chip::ErrorStr(err)));
 
+#if CONFIG_USE_SEPARATE_EVENTLOOP
```

Comment:
Since this code is evolving into a demonstration, could you add a comment here that `ServiceEvents()` calls `StartEventLoopTask()`? Otherwise the naive reader sees an unpaired `StopEventLoopTask()` and it won't be obvious where it starts. (Maybe ServiceEvents shouldn't, or shouldn't exist, but that's another story.)

Reply:
Added a comment.